sensor_msgs/Image img
Detection_results det_ress
sensor_msgs/PointCloud2 pcl
---
Detection_results detResultWithPosition